client/src/components/questionsAnswers/Qna.jsx

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable import/extensions */
+/* eslint-disable no-unused-vars */
+import React from 'react';
+import axios from 'axios';
+import Search from './Search.jsx';
+import Accordion from './Accordion.jsx';
+
+const { useState, useEffect } = React;
+
+function Qna(props) {
+  const [search, setSearch] = useState('');
+  const [questions, setQuestions] = useState([]);
+  // const { product } = props;
```

Comment:
Is this supposed to be commented out or omitted? If it is supposed to be commented out, please provide context within the comment. If not, please omit!